src/components/sbb-divider/sbb-divider.scss

```diff
@@ -0,0 +1,25 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --divider-color: var(--sbb-color-cloud-default);
+  --border-width: var(--sbb-border-width-1x);
+}
+
+:host(.sbb-divider) {
```

Comment:
This is not necessary as it is provided by `@include host-component-properties;`

Reply:
Yep 👍